diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -0,0 +1,239 @@
+function StudentQuiz(runtime, element) {
```

Comment:
Would be good if you can add `"use strict";` here and make sure this complies with strict mode.